packages/web-components/src/components/expressive-modal/expressive-modal.ts

```diff
@@ -290,7 +312,7 @@ class DDSExpressiveModal extends StableSelectorMixin(HostListenerMixin(LitElemen
   protected _renderHeader(): TemplateResult | SVGTemplateResult | void {
     const { _hasHeader: hasHeader, _hasBody: hasBody, _hasFooter: hasFooter } = this;
     const headerClasses = classMap({
-      [`${ddsPrefix}-ce--modal__hedaer--with-body`]: hasHeader && (hasBody || hasFooter),
+      [`${ddsPrefix}-ce--modal__header--with-body`]: hasHeader && (hasBody || hasFooter),
```

Comment:
@kennylam this is changed here already - do we need to maintain the old class here as well?

Reply:
Gotcha, I missed that change. As long as we match all the component names with CSS changes we should be good. 👍